src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -949,3 +950,13 @@ def UpdatePermission(func):  # noqa: N802
     """
     _set_func_permissions(func, Permission.UPDATE)
     return func
+
+
+# Empty request body decorator
+
+def allow_empty_body(func):  # noqa: N802
+    """
+    adsfsdfsdf.
+    """
+    func._cp_config = {'tools.json_in.force': False}
```

Comment:
I suggest to append that setting here because it may happen that another decorator or code has already set the `func._cp_config` dict.

Reply:
sorry I didn't get which setting you implying to append?

Reply:
```suggestion
    if not isinstance(func._cp_config, dict):
        func._cp_config = {}
    func._cp_config.update({'tools.json_in.force': False})
```

Reply:
But I think, just updating the config will work too..!?
func._cp_config.update({'tools.json_in.force': False})

Reply:
What if `func._cp_config` has not yet been created? You could take the Python approach:
```py
try:
  func._cp_config['tools.json_in.force] =  False
except AttributeError, KeyError:
  func._cp_config = {'tools.json_in.force': False}
```